eslint.json

```diff
@@ -27,12 +30,19 @@
         "SwitchCase": 1
       }
     ],
+    "jsdoc/check-param-names": 1,
+    "jsdoc/check-tag-names": 1,
+    "jsdoc/newline-after-description": 1,
+    "jsdoc/require-description-complete-sentence": 1,
+    "jsdoc/require-param": 1,
+    "jsdoc/require-param-name": 1,
+    "jsdoc/require-param-type": 1,
+    "jsdoc/valid-types": 1,
     "key-spacing": "error",
     "linebreak-style": [
       "error",
       "unix"
     ],
-    "newline-before-return": "error",
```

Comment:
Removed?

Reply:
Yes it's replaced with "padding-line-between-statements" key. https://eslint.org/docs/rules/newline-before-return

Reply:
I've added it myself.

Reply:
Oh Okay.